packages/drivers/odsp-driver/src/epochTracker.ts

```diff
@@ -10,39 +10,111 @@ import { ThrottlingError } from "@fluidframework/driver-utils";
 import { IConnected } from "@fluidframework/protocol-definitions";
 import { PerformanceEvent, LoggingError } from "@fluidframework/telemetry-utils";
 import { fetchAndParseAsJSONHelper, fetchArray, IOdspResponse } from "./odspUtils";
-import { ICacheEntry, IFileEntry, LocalPersistentCacheAdapter } from "./odspCache";
+import {
+    ICacheEntry,
+    IEntry,
+    IFileEntry,
+    IOdspCache,
+    IPersistedCache,
+    INonPersistentCache,
+    LocalPersistentCache,
+    IPersistedFileCache,
+    snapshotKey,
+ } from "./odspCache";
 import { RateLimiter } from "./rateLimiter";
 
-export type FetchType = "blob" | "createBlob" | "createFile" | "joinSession" | "ops" | "other" | "snapshotTree" |
+export type FetchType = "blob" | "createBlob" | "createFile" | "joinSession" | "ops" | "test" | "snapshotTree" |
     "treesLatest" | "uploadSummary" | "push" | "versions";
 
+export type FetchTypeInternal = FetchType | "cache";
+
+// exported only of test purposes
+export interface IVersionedValueWithEpoch<T> {
+    value: T;
+    fluidEpoch: string,
+    version: "0.2",
+}
+
+// exported only of test purposes
+export const persistedCacheValueVersion = "0.2";
```

Comment:
integers are a lot easier to work with, do we need the complexity of a semanitic version here?